var/spack/repos/builtin/packages/isaac-server/package.py

```diff
@@ -29,7 +31,8 @@ class IsaacServer(CMakePackage):
 
     depends_on('cmake@3.3:', type='build')
     depends_on('jpeg', type='link')
-    depends_on('jansson@:2.9', type='link')
+    depends_on('jansson@:2.9', type='link', when='@:1.5.1')
+    depends_on('jansson', type='link', when='@1.5.2:')
```

Comment:
This can be simplified as:
```suggestion
    depends_on('jansson', type='link')
```

Reply:
Thanks, I was not sure if this is equivalent, I changed it to the suggested version.

Reply:
No worries, this is equivalent. The directive states that `isaac-server` has an unconditional dependency on `jansson`. The directive above adds more constraints if the version of `isaac-server` is 1.5.1 or less.